src/main.js

```diff
@@ -224,10 +224,11 @@ $(document).ready(() => {
      * @param {string} newKeys
      * @param {string?} oldKeys
      */
-    function handleHotKeys(newKeys, oldKeys) {
+    function updateHotkey(newKeys, oldKeys) {
+      key.filter = () => $sidebar.is(':visible');
```

Comment:
I might be mistaken. But when the sidebar is closed, is it still "visible"?

Reply:
Yes, it's still visible outside the left bound of the screen.